installer/cluster.go

```diff
@@ -578,8 +578,12 @@ iptables -A FORWARD -i eth0 -j DROP
 
 var startScript = template.Must(template.New("start.sh").Parse(`
 #!/bin/sh
+set -e -x
+
 FIRST_BOOT="/var/lib/flynn/first-boot"
-if [[ ! -f "${FIRST_BOOT}" ]]; then
+mkdir -p /var/lib/flynn
```

Comment:
We should probably do this in the install script too, no issue with it being here though.